### ISSUE TITLE
Stuck on boot step systemd-update-utmp-runlevel.service on Pi 4

### ISSUE THREAD

**Issue by IvoKara** [ptrsr/pi-ci#14]
Firstly I wanted to thank you, this project is amazing!

I have tried several ways to do this but the end result is the same.

I have successfully configured my custom `distro.img` image ready to be flashed on SD card.

After a successful flash, everything looks promising until I plug the SD card on my _Raspberry Pi 4_.

It boots without any problems until it reaches this point and it just freezes and do not proceed further:
```
[  OK  ] Finished systemd-update-utmp-runlevel.service - Record Runlevel Change in UTMP.
```

I have tried several ways to flash the image:
* With different way of flashing
    - by exporting and flashing with `balenaEtcher`
    - by exporting and flashing with `Pi-Imager`
    - by flashing directly with `docker rum --rm -it ptrsr/pi-ci flash` command on a linux machine
    - by flashing with `dd` 
* SD is not corrupted have booted the stock Raspberry Pi OS bookworm lite without problem

Any clarifications would be helpful.

Tkanks!

**Comment by ptrsr** [comment]
Hi @IvoKara , thanks for your kind words and interest in the project!

It could be that after the login prompt is reached (on the TTY), additional services reach a "OK" state and overwrite the prompt. Could you try typing something to see if the prompt shows up again?

Thanks,
R.

**Comment by IvoKara** [comment]
Yep, what I actually did and worked is to copy the `override.conf` from the `serial-getty@ttyAMA0.service` configuration file and put it inside `getty@tty1.service.d` directory in `/etc/systemd/system`

```bash
cd /etc/systemd/system/
cp  serial-getty@ttyAMA0.service.d/override.conf getty@tty1.service.d/override.conf
```

As I believe that you set up `/dev/ttyAMA0` in order to be able to log in into the VM from the emulator.

In fact everything was working, nothing was freezed or stuck, because when my Raspberry Pi booted I pressed <kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>F3</kbd> which opened `/dev/tty3` and then I was able to log in.

**Comment by ptrsr** [comment]
Aaah you're right. When upgrading to bookworm, I noticed that the login prompt was changed from `ttyAMA0` to `tty1`. For the emulator (QEMU) this is fine, but for the real Pi this may not work anymore with the override.

I'll have a look in the next couple weeks, thanks for pointing this out!

**Comment by ptrsr** [comment]
# Story
Fix the login prompt (`getty`) on `tty1`.

## Test
**GIVEN** the default distro image included in PI-CI
**WHEN** the default image is flashed on an SD card and started using a physical RPi
**THEN** the user should be able to log in using the default TTY (`tty1`).

## Current behavior
- TODO (needs further testing)
